README.md

```diff
@@ -12,6 +12,18 @@ https://developer.webex.com/docs/login-with-webex
 
 Try it out by chosing one of the available flows: https://webexsamples.github.io/login-with-webex
 
+To start the server run:
+
+`npm start`
+
+Optional parameters are:
+WEBEX_BASE=https://integration.webexapis.com
```

Comment:
Please change https://integration.webexapis.com to https://webexapis.com according to the [docs](https://developer.webex.com/docs/api/v1/authorizations) for non-cisco users.

Reply:
I also noticed that the authorization code flow ends by displaying the authorization code. Is this the intended end of the flow? should we add functionality that shows user info?
![Screenshot 2024-10-22 at 2 07 52 PM](https://github.com/user-attachments/assets/4fe97522-7304-4436-8aa4-e2474db35e5a)

Reply:
Sure can change the example. For my use case it was helpful to have the code unformatted, feel free to change this how you see fit if you think there's something more generically useful